src/components/apps/savedLaunch/SavedLaunchListing.js

```diff
@@ -29,23 +33,33 @@ import {
     listSavedLaunches,
     deleteSavedLaunch,
 } from "serviceFacades/savedLaunches";
+import {
+    ALL_INSTANT_LAUNCHES_KEY,
+    listFullInstantLaunches,
+    addInstantLaunch,
+} from "serviceFacades/instantlaunches";
+
 import { useConfig } from "contexts/config";
 import { useUserProfile } from "contexts/userProfile";
 
 import buildID from "components/utils/DebugIDUtil";
 import CopyTextArea from "components/copy/CopyTextArea";
 
+import Button from "@mui/material/Button";
```

Comment:
I don't know if it would be good to change these all into a single import (same with the icons). I just copied the code style within this file, though.

Reply:
Cool. Wasn't sure if we had any sort of code style we cared about, but always very glad to just match existing code :grin: